src/components/Feed/WrapToken.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react'
+import styled from 'styled-components'
+import {
+  Box,
+  Button,
+  GU,
+  Help,
+  useLayout,
+  textStyle,
+  useTheme,
+} from '@1hive/1hive-ui'
+
+import { useAppState } from '../../providers/AppState'
+
+import { formatTokenAmount } from '../../utils/token-utils'
+
+import wrappedIcon from '../../assets/wrappedIcon.svg'
+import unwrappedIcon from '../../assets/unwrappedIcon.svg'
+
+function WrapToken({ onUnwrapToken, onWrapToken }) {
+  const { layoutName } = useLayout()
+  const {
+    accountBalance: gardenTokenBalance,
+    stakeToken,
+    wrappableAccountBalance,
+    wrappableToken,
+  } = useAppState()
+
+  const theme = useTheme()
+  const compactMode = layoutName === 'small' || layoutName === 'medium'
+  const tabletMode = layoutName === 'medium'
+
+  return (
+    <Box
+      css={`
+        ${tabletMode && `height: 100%;`}
+        ${!compactMode && `margin-bottom: ${3 * GU}px;`}
+      `}
+    >
+      <div
+        css={`
+          display: flex;
+          flex-direction: row;
+          justify-content: center;
+        `}
+      >
+        <Token
+          mode="wrap"
+          balance={wrappableAccountBalance}
+          token={wrappableToken}
+          onClick={onWrapToken}
+        />
+        <LineSeparator border={theme.border} />
+        <Token
+          mode="unwrap"
+          balance={gardenTokenBalance}
+          token={stakeToken}
+          onClick={onUnwrapToken}
+        />
+      </div>
+    </Box>
+  )
+}
+
+function Token({ mode, balance, token, onClick }) {
+  const wrapMode = mode === 'wrap'
+  const icon = wrapMode ? unwrappedIcon : wrappedIcon
+  const button = wrapMode
+    ? { mode: 'strong', label: 'Wrap' }
+    : { mode: 'normal', label: 'Unwrap' }
+
+  return (
+    <div
+      css={`
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        width: 100%;
+        ${textStyle('body2')};
+      `}
+    >
+      <img src={icon} height="48" width="48" />
+      <span
+        css={`
+          font-weight: 400;
```

Comment:
Let's use `bold` as we don't have a font with that font-weight

```suggestion
          font-weight: bold;
```